app/controllers/api/samples_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Api::SamplesController < ApplicationController
+    before_action :set_user, only: [:show, :update]
+    
+    def index
+        @users = User.all
+
+        render json: @users
```

Comment:
We don't need this empty line. https://rubystyle.guide/#empty-lines-around-bodies

```suggestion
        @users = User.all
        render json: @users
```